### ISSUE TITLE
Another issue(emmm.....i can't open another issue in the other repository.)

### ISSUE THREAD

**Issue by xinqingbuhao** [gdahia/fingerprint-pore-detection#1]
In inference process, use whole image means that each pixel on the entire image has to be framed by a 17*17 box(17*17 patch) .then these patches will be the input of net? finally we will get a pore probability map（size：320-17，240-17）to apply NMS (each pixel>pt  be framed by a 7*7 bounding box) . right?

emmm...another issue : the input size of net is 17x17 ,then output size is 17-16,17-16(use yours FCN) means that final result logits is a single number . right? 

thx .
